.github/labeler.yml

```diff
@@ -0,0 +1,39 @@
+# .github/labeler.yml
+# Add 'root' label to any root file changes
+# Quotation marks are required for the leading asterisk
+root:
+- changed-files:
+  - any-glob-to-any-file: '*'
+
+# Add 'AnyChange' label to any changes within the entire repository
+AnyChange:
+- changed-files:
+  - any-glob-to-any-file: '**'
+
+# file filters
+Standard-Change:
+- changed-files:
+  - any-glob-to-any-file: '.ci/**'
```

Comment:
This needs to be adopted to the contents of the daml repo:
  - any-glob-to-any-file: '.ci/**' => ci and sdk/ci
  - any-glob-to-any-file: 'release/**' => sdk/release
  - any-glob-to-any-file: 'deployment/**' =>devnull
  - any-glob-to-any-file: 'scripts/ci/**' => sdk/build-scripts
  - any-glob-to-any-file: '.ci/Dockerfile' => devnull
  - any-glob-to-any-file: 'nix/**' => sdk/nix
  - any-glob-to-any-file: 'shell.nix' => sdk/shell.nix
  - any-glob-to-any-file: 'project/**' => all the BUILD.bazel files
  - any-glob-to-any-file: 'build.sbt' => sdk/bazel_tools
  - any-glob-to-any-file: 'dependencies.json' => all *deps/bzl files

Likely there will be more

Reply:
@acochard-da @dylant-da @paulbrauner-da
The question is which of the files when touched can potentially result in a change to the the way any of the deliverables is compiled/built